hub/admin.py

```diff
@@ -41,9 +41,9 @@ def delete_queryset(self, request, queryset):
             # Maybe it still makes sense because KPI users are deleted.
             self.message_user(
                 request,
-                _('Could not delete users in KoBoCAT database. They may own '
-                  'projects and/or submissions. Log into KoBoCAT admin '
-                  'interface and delete them from there.'),
+                'Could not delete users in KoBoCAT database. They may own '
+                'projects and/or submissions. Log into KoBoCAT admin '
+                'interface and delete them from there.',
```

Comment:
Can we verify that `gettext()` imports are still needed?

Reply:
removed